common/src/main/java/whocraft/tardis_refined/common/tardis/themes/DesktopTheme.java

```diff
@@ -4,25 +4,36 @@
 import net.minecraft.resources.ResourceLocation;
 import whocraft.tardis_refined.constants.ModMessages;
 
+import java.util.Optional;
+
 
 public class DesktopTheme {
 
     private final ResourceLocation uiTexture;
     public boolean availableByDefault = false;
-    public String id = "";
+    public ResourceLocation identifier;
+    public String name = "";
     public ResourceLocation location;
```

Comment:
Would be good to get some getters and setters for these fields rather than make all of them public.